js/components/widgets/Title.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @property {number} priority
+ * @property {string} content
+ */
+
+/**
+ * @function Title
+ * @typedef {Object} Title
```

Comment:
???

Reply:
Why are there two comments?

(I did not fully understand the dependence, what comes from)